.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </state>
+</component>
```

Comment:
As you see all idea's settings are in the repo. If you want to exclude it you have to delete settings, then commit, then add again settings

Reply:
done